src/Manipulator/ServicesManipulator.php

```diff
@@ -59,7 +59,7 @@ public function addResource($serviceId, $modelClass, $adminClass, $controllerNam
         $code = "services:\n";
 
         if (is_file($this->file)) {
-            $code = rtrim(file_get_contents($this->file));
+            $code = rtrim(file_get_contents($this->file) ?: '');
```

Comment:
maybe better ??

Reply:
file_get_contents($this->file) can return `false`, that the value which lead to an error.

Reply:
I don't know what the possible values for `$this->file` are, but I think we could make this check more explicit.
`?:` will forward the execution flow to the else part for values like `0`.

Reply:
Done, I throw an exception if false is returned.